api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWithMdsEvaluationCohortQueries.java

```diff
@@ -0,0 +1,5077 @@
+package org.openmrs.module.eptsreports.reporting.library.cohorts;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.text.StringSubstitutor;
+import org.openmrs.Location;
+import org.openmrs.module.eptsreports.metadata.CommonMetadata;
+import org.openmrs.module.eptsreports.metadata.HivMetadata;
+import org.openmrs.module.eptsreports.metadata.TbMetadata;
+import org.openmrs.module.eptsreports.reporting.library.queries.ListOfPatientsWithMdsEvaluationQueries;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.SqlCohortDefinition;
+import org.openmrs.module.reporting.data.DataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.SqlPatientDataDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ListOfPatientsWithMdsEvaluationCohortQueries {
+
+  private final HivMetadata hivMetadata;
+
+  private final TbMetadata tbMetadata;
+
+  private final CommonMetadata commonMetadata;
+
+  String inclusionStartMonthAndDay = "'-12-21'";
+  String inclusionEndMonthAndDay = "'-06-20'";
+
+  @Autowired
+  public ListOfPatientsWithMdsEvaluationCohortQueries(
+      HivMetadata hivMetadata, TbMetadata tbMetadata, CommonMetadata commonMetadata) {
+    this.hivMetadata = hivMetadata;
+    this.tbMetadata = tbMetadata;
+    this.commonMetadata = commonMetadata;
+  }
+
+  /**
+   * <b>RF9 - Relatório – Informação do utente - Secção A1 a A9</b>
+   *
+   * <p>A2- Coorte: (coluna B) – Resposta = 12 meses, caso o utente tenha iniciado TARV na coorte de
+   * 12 meses, ou Resposta = 24 meses, caso o utente tenha iniciado TARV na coorte de 24 meses
+   * (RF4).
+   *
+   * @return {CohortDefinition}
+   */
+  public CohortDefinition getPatientsInitiatedART12Or24Months(
+      int numberOfYearsStartDateFor12,
+      int numberOfYearsEndDateFor12,
+      int numberOfYearsStartDateFor24,
+      int numberOfYearsEndDateFor24) {
```

Comment:
These parameters are all necessary because we have 2 Cohorts 
12 Month Cohort  = 
Data Início Inclusão = “21 de Dezembro” de “Ano de Avaliação” menos (-) 2 anos [numberOfYearsStartDateFor12]
Data Fim Inclusão = “20 de Junho” de “Ano de Avaliação” menos (-) 1 ano [numberOfYearsEndDateFor12]

24 Months Cohort =
Data Início Inclusão = “21 de Dezembro” de “Ano de Avaliação” menos (-) 3 anos [numberOfYearsStartDateFor24]
Data Fim Inclusão = “20 de Junho” de “Ano de Avaliação” menos (-) 2 ano [numberOfYearsEndDateFor24]

which are then calculated accordingly within the CohortDefinition (i.e. row filter/base cohort) query.